### PR TITLE
Reporter now uses sqs queue to determine which crawls to process/report

### PR DESCRIPTION
**Pull request by SourenP**
note: report.js is no longer necessary

### CODE REVIEW

**Comment by SourenP** [comment]
note: report.js is no longer necessary

File: index.js [hunk 1]
```diff
@@ -8,22 +8,18 @@ commander
...
-  .action(function(startId, endId, dbUrl, graphiteUrl) {
-    src.report(startId, endId, dbUrl, graphiteUrl);
+  .action(function(max, timeout) {
+    var queueUrl = process.env.SQS_URL;
+    var dbUrl = process.env.DATABASE_URL;
+    var graphiteUrl = process.env.GRAPHITE_URL;
+    if (queueUrl && dbUrl && graphiteUrl) {
+      src.live(max, timeout, queueUrl, dbUrl, graphiteUrl);
+    } else {
+      console.error("Missing enviornment variable.")
```
Comment (wilsonianb) [inline index.js]: `enviornment` -> `environment`

Comment (SourenP) [inline index.js]: lol

File: src/lib/report_graphite.js [hunk 2]
```diff
@@ -2,31 +2,29 @@ var rc_util = require('rippled-network-crawler/src/lib/utility.js');
...
-      }
-    });
-  }
+  });
+  graphiteClient.write(metrics, moment(crawl.start_at).valueOf(), function(error) {
+    if (error) {
+      console.error(error);
+    } else {
+    console.log('Wrote crawl %d \t at %s \t to graphite',
+            crawl.id, moment().format());
```
Comment (wilsonianb) [inline src/lib/report_graphite.js]: Indentation

File: index.js (hunk 1 above)
Comment (SourenP) [inline index.js]: Now we know how meticulous you are during code review :) :+1:

File: src/live.js [hunk 3]
```diff
@@ -1,69 +1,61 @@
...
+          if (!crawl) {
+            console.error('No crawls with id %s', id);
+            count -= 1;
+            return -1;
+          }
+
+          processCrawl(crawl);
+          count -= 1;
+
+        }).then(function() {
```
Comment (wilsonianb) [inline src/live.js]: Is there a scenario where `processCrawl` fails and we wouldn't want to delete the message?

Comment (SourenP) [inline src/live.js]: good point
I guess it should crash and never reach the line that deletes

Comment (wilsonianb) [inline src/live.js]: I think you could change this to:
```
  return processCrawl(crawl);
}).then(function() {
  count-=1;
```

**Comment by SourenP** [comment]
lol, this is just useless processing now but the code will be useful later :P

**Comment by wilsonianb** [comment]
So this will stop reporting to graphite?

**Comment by SourenP** [comment]
Yes
There is no point in writing that code since the structure will change anyways.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**